net/node/node.go

```diff
@@ -28,6 +28,7 @@ import (
 
 const (
 	MaxSyncHeaderReq     = 2                // max concurrent sync header request count
+	MaxMsgChanNum        = 2048             // max goroutine num for message handler
```

Comment:
Is 2048 too small?

Reply:
I tested goroutine performance just now. Creating 20480 goroutines and do a simple memory operation(1K memory) in each goroutine will cost 100M memory. Will change it to 20480.